Mathlib/Tactic/Linter/DirectoryDependency.lean

```diff
@@ -220,6 +220,43 @@ def allowedImportDirs : NamePrefixRel := .ofArray #[
   (`Mathlib.Tactic.Linter, `ImportGraph),
   (`Mathlib.Tactic.Linter, `Mathlib.Tactic.MinImports),
   (`Mathlib.Tactic.Linter.TextBased, `Mathlib.Data.Nat.Notation),
+
+  (`Mathlib.Logic, `Batteries),
+  -- XXX: should this import be allowed on a more fine-grained level?
+  (`Mathlib.Logic, `Mathlib.Control),
+  (`Mathlib.Logic, `Mathlib.Lean),
+  (`Mathlib.Logic, `Mathlib.Util),
+  (`Mathlib.Logic, `Mathlib.Tactic),
+  (`Mathlib.Logic.Fin.Rotate, `Mathlib.Algebra.Group.Fin.Basic),
+  (`Mathlib.Logic.Hydra, `Mathlib.GroupTheory),
+  (`Mathlib.Logic, `Mathlib.Algebra.Notation),
+  (`Mathlib.Logic, `Mathlib.Algebra.NeZero),
+  -- XXX: should this import be allowed only on a more fine-grained level (or at all)?
+  (`Mathlib.Logic, `Mathlib.Data),
```

Comment:
I think importing Data is fine here. We have a lot of definitions of embeddings/equivalences for specific objects, living in Mathlib.Logic.
```suggestion
  (`Mathlib.Logic, `Mathlib.Data),
```